src/libs/ajax/User.ts

```diff
@@ -116,9 +157,10 @@ export const User = {
     const res = await axios.post(url, keys, Config.authOpts());
     return res.data;
   },
-  getApprovedDatasets: async () => {
+
+  getApprovedDatasets: async (): Promise<ApprovedDataset> => {
```

Comment:
Should this also be returning a list?
```suggestion
  getApprovedDatasets: async (): Promise<ApprovedDataset[]> => {
```

Reply:
Yep, I missed this one too 👍